content/projects/nodejs/ajax/_index.md

```diff
@@ -45,10 +45,14 @@ Also, make sure that if you create any new visitors then they are visible on the
 
 ## Add a visitor
 
-Add a button labelled "New Visitor". This button should redirect the user to the new visitor form that was created in an earlier step in this project. 
+Add a button labelled "New Visitor". This button should redirect the user to the new visitor form that was created in an earlier step in this project.
 
 Update the "thank you page" so that it only displays for 3 seconds and then redirects the user back to the visitors table. Of course the new visiter should be visible in the table.
 
+## Up for a challenge?
+
+So far, you've made use of only 2 API end points(`listAllVisitors` and `deleteVisitor`). Can you figure out how to make use the `updateVisitor` endpoint? The implementation here is completely up to you.
```

Comment:
- The endpoints you are referencing here were changed to follow "RESTful API design" a while back. [See](http://syllabus.africacode.net/projects/nodejs/api/index.html)